draft-ietf-scitt-software-use-cases.md

```diff
@@ -298,35 +297,37 @@ A consumer of a released software wants:
 There is no standardized way to:
 
 - provide an authenticity proof that can be checked by off-line systems in a simple and uniform fashion
-- enable rich systems, regular systems, and constrained systems to conduct authenticity checks via the same procedure / code base
-- to verify the authenticity and integrity of software in a fashion that scales from applications such as global open source repositories down to off-line constrained devices
+- enable high performance, and constrained systems to conduct authenticity checks
+- verify the authenticity and integrity of software in a fashion that scales
 
 ## Firmware Delivery to Large Set of Devices
 
-Firmware is a critical component for successful execution of any device, constrained IoT device or general purpose computer.
-It is often the bedrock on which the security story of the devices it powers.
+Firmware is a critical component of constrained IoT devices and general purpose computers.
```

Comment:
Minor, non-blocking: I am cool with this but generalized it because I felt even consumer laptops fall in this category with UEFI initialization as a common use case that is like IoT but not IoT.

Reply:
yeah, as I said on the list, I feel these use cases need more than just phrasing updates... I am not going to attempt that in this PR.

Reply:
Agreed, you can disregard this one (and the other, since it seems I cannot resolve the comment).